components/app-layouts/providers.tsx

```diff
@@ -65,6 +65,11 @@ export const DashboardThirdwebProvider: ComponentWithChildren = ({
       }}
       storageInterface={StorageSingleton}
       walletConnectors={walletConnectors}
+      authConfig={{
+        authUrl: "/api/auth",
+        domain: "thirdweb.com",
+        loginRedirect: "/faucet/mumbai",
+      }}
```

Comment:
auth is a whole different discussion I don't think we're ready to implement auth on the dashboard yet